src/components/editTestCase/qdm/LeftPanel/ElementsTab/Elements/DataElementsCard/attributes/DisplayAttributeInputs.tsx

```diff
@@ -50,6 +51,27 @@ const DisplayAttributeInputs = ({
             }}
           />
         );
+      case "DateTime":
+        return (
+          <DateTimeInput
+            label="DateTime"
+            canEdit={true}
+            dateTime={null}
+            attributeName="DateTime"
+            onDateTimeChange={(e) => {
+              const newDateTime = dayjs.utc(e);
+              const newCQLDateTime: CQL.Date = new CQL.DateTime(
+                newDateTime.year(),
+                newDateTime.month() + 1,
+                newDateTime.date(),
+                newDateTime.get("hour"),
+                newDateTime.get("minute"),
+                0
+              );
+              setAttributeValue(newCQLDateTime);
+            }}
```

Comment:
I think the conversion is already done in `DateTimeInput`. This will be enough I think `onDateTimeChange={(value, name) => setAttributeValue(value)}`

Reply:
Oops, forgot to hit save before I committed! Thank you